src/proxy-page/steps/fixLinks.ts

```diff
@@ -55,13 +55,13 @@ export default (config: ConfigService): Step => {
     // For direct Url and Uri we look for two patterns
     // $1 the domain to remove and $2 the rest of the URL to keep
     const searchUrl = new RegExp(`(https?://${domain}/wiki)(.*)`);
-    const searchUri = new RegExp(`(\/wiki)(.*)`);
+    const searchUri = new RegExp(`^(\/wiki)(.*)`);
     // For Url and Uri with anchor we look for four patterns
     // $1 the domain to remove, $2 the path of the pag, $3 the title and $4 the heading achor
     const searchUrlwithAnchor = new RegExp(
       `(https?://${domain}/wiki)(.*\/)(.*)#(.*)`,
     );
-    const searchUriwithAnchor = new RegExp(`(\/wiki)(.*\/)(.*)#(.*)`);
+    const searchUriwithAnchor = new RegExp(`^(\/wiki)(.*\/)(.*)#(.*)`);
```

Comment:
Perhaps it would be interesting to store the prefix `/wiki` in a variable so we could use it both here and at line 57-58 to avoid repetition.

Reply:
There are even more occurrences of those strings, and I think it will even cost the same of changing this string than changing the constant, in the unlikely event Atlassian change this prefix.
I won't take the risk to make multiple changes in this critical routine in konviw (probably the most critical of the full software package). My main point whether you see the new regular expression fine.